src/components/shared/Validations.tsx

```diff
@@ -22,9 +24,17 @@ export const Validations: React.FunctionComponent<IValidations> = ({
     </div>
   );
 
+  const visibility =
+    viewMode !== 'standalone' || (readOnly && writeOnly) ? null : readOnly ? (
+      <span className="ml-2 text-darken-7 dark:text-lighten-6">read-only</span>
+    ) : writeOnly ? (
+      <span className="ml-2 text-darken-7 dark:text-lighten-6">write-only</span>
+    ) : null;
```

Comment:
Generally it does two things:
- show those validations only in standalone mode
- if some someone puts both `readOnly` and `writeOnly` (which doesn't make sense) show nothing

Perhaps it could be cleaned for readability, or commented

Reply:
Oooh, now it makes sense. I was lost in those triple nested ternaries. 😄 

Can you - as you said - either comment it, or write it in a more verbose way please? (`let` and `if/else` is fine in this case I guess)

Reply:
Just had an idea on how I would maybe write this. Instead of a single `const visibility...`
```tsx
const writeOnlyFlag = viewMode === 'standalone' && writeOnly ? (<span className="ml-2 text-darken-7 dark:text-lighten-6">write-only</span>) : null;
const readOnlyFlag = viewMode === 'standalone' && readOnly ? (<span className="ml-2 text-darken-7 dark:text-lighten-6">read-only</span>) : null;
```
or another `&&` instead of the ternary if you prefer.
Then in the JSX just
```jsx
{readOnlyFlag}
{writeOnlyFlag}
```


But anything else works too, including a comment.

Reply:
I just saw this comment after pushing the change with simplifying it. Your suggestion looks quite good though so I might replace it

Reply:
But actually it misses the case when they are both present so needs one more check